airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -272,34 +275,43 @@ def request_params(self, stream_state: Mapping[str, Any], stream_slice: Mapping[
         return params
 
 
-class Repositories(GithubStream):
+# Below are full refresh streams
+
+
+class Assignees(GithubStream):
     """
-    This stream is technical and not intended for the user, it is used only to obtain repositories for organizations.
-    API docs: https://docs.github.com/en/rest/reference/repos#list-organization-repositories
+    API docs: https://docs.github.com/en/rest/reference/issues#list-assignees
     """
 
-    def __init__(self, organizations: List[str], **kwargs):
-        super(GithubStream, self).__init__(**kwargs)
-        self.organizations = organizations
 
-    def stream_slices(self, **kwargs) -> Iterable[Optional[Mapping[str, any]]]:
-        for organization in self.organizations:
-            yield {"organization": organization}
+class PullRequestStats(GithubStream):
+    """
+    API docs: https://docs.github.com/en/rest/reference/pulls#get-a-pull-request
+    """
 
-    def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
-        return f"orgs/{stream_slice['organization']}/repos"
+    fields_to_minimize = ("merged_by",)
```

Comment:
same question as below about minimization

Reply:
Yea we can keep it un-minimized.